apps/dg/components/case_card/column_resize_handle.js

```diff
@@ -0,0 +1,146 @@
+       *  enabled: {boolean} - enable/disable resizing
+       *  containerWidth: {number} - width of table in pixels
+       *  columnWidth: {number} - width of column to resize in pixels
+       *  minWidth: {number} - minimum width of column in pixels
+       *  maxWidth: {number} - maximum width of column in pixels
+       *  onResize(width: number): {function} - Called with updated value of width property
+       */
+      var resizeTouchId = null,
+          deltaStart = null;
+
+      function stop(e) {
+        e.preventDefault();
+        e.stopPropagation();
+      }
+
+      return {
+
+        getInitialState: function() {
+          return {
+            isResizing: false,
+            resizeWidth: null
+          };
+        },
+
+        componentDidMount: function() {
+          this.componentDidRender();
+        },
+
+        componentDidUpdate: function() {
+          this.componentDidRender();
+        },
+
+        componentDidRender: function() {
+          if (this.state.resizeWidth !== this.props.columnWidth)
+            this.setState({ resizeWidth: this.props.columnWidth });
+        },
+
+        render: function() {
+          var style = this.state.resizeWidth != null
+                        ? { left: this.state.resizeWidth }
+                        : null;
+          return (this.props.enabled || this.state.isResizing) &&
+            div({
+              className: this.props.className + ' column-resize-handle dg-wants-mouse dg-wants-touch',
+              onMouseDown: this.handleMouseDown,
+              onTouchStart: this.handleInitialTouch,
+              style: style
+            },
+            div({
+              className: "column-resize-handle-render" +
+                          (this.state.isResizing ? " is-resizing" : "")
+            }));
+        },
+
+        beginResize: function(e) {
+          deltaStart = e.pageX - this.state.resizeWidth;
+          this.setState({ isResizing: true });
+        },
+
+        continueResize: function(e) {
+          var newWidth = e.pageX - deltaStart,
+              minWidth = this.props.minWidth || 20,
+              maxWidth = this.props.containerWidth - minWidth;
+          newWidth = Math.max(newWidth, minWidth);
+          newWidth = Math.min(newWidth, maxWidth);
+          deltaStart = e.pageX - newWidth;
+          if (newWidth !== this.state.resizeWidth) {
+            this.props.onResize && this.props.onResize(newWidth);
+            this.setState({ resizeWidth: newWidth });
+          }
+        },
+
+        endResize: function() {
+          this.setState({ isResizing: false });
+        },
+
+        handleMouseDown: function(e) {
+          if (!this.props.enabled || (e.button !== 0)) return;
+          this.beginResize(e);
+          document.addEventListener('mousemove', this.handleMouseMove, true);
+          document.addEventListener('mouseup', this.handleMouseUp, true);
```

Comment:
With both mouse and touch events, we install the handlers we need precisely when we need them. This contrasts with the SproutCore/SC.RootResponder approach of installing event handlers centrally and having all events captured by a single entity that then must keep track of a lot of state to determine how/where to dispatch any given event.